### PR TITLE
Feature/docker flor

### PR DESCRIPTION
**Pull request by florpadilla**
Hi Luis, please review this pipeline. I modified the Jenkinsfile including the "Continuous Integration" and the "Continuous deployment" and the ports in the dockerfile.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -55,6 +53,60 @@ pipeline {
                 waitForQualityGate abortPipeline: true
             }
         }
+        stage('Upload to docker hub'){
```
Comment (luisdavidparra) [inline Jenkinsfile:56]: You should document all the stages

File: Jenkinsfile [hunk 2]
```diff
@@ -55,6 +53,60 @@ pipeline {
...
+            }
+            post {
+                always {
+                    script {
+                        sh 'docker logout'
+                    }
+                }
+            }
+        }
+        //aqu termina Continuous Integration
```
Comment (luisdavidparra) [inline Jenkinsfile]: This comments must be in english

**Review by luisdavidparra — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (florpadilla) [reply Jenkinsfile]: Thanks! Done

**Comment by luisdavidparra** [comment]
Looks good Maria!

**Pull request merged**